scipy/linalg/_basic.py

```diff
@@ -264,14 +264,14 @@ def solve(a, b, lower=False, overwrite_a=False,
 def solve_triangular(a, b, trans=0, lower=False, unit_diagonal=False,
                      overwrite_b=False, check_finite=True):
     """
-    Solve the equation `a x = b` for `x`, assuming a is a triangular matrix.
+    Solve the equation ``a x = b`` for `x`, assuming a is a triangular matrix.
```

Comment:
should this not be in Math notation?

Reply:
I think it is obvious enough, everyone ends up here knows what `a x = b` is. Math notation is just clutter for three chars in my opinion.

Reply:
I think there's a case for `a @ x == b`, but I agree that it will be understood as-is. In any case, this is better than it was.

I noticed that `x` is not a "parameter" per se, rather it is the character used to denote the returned value. I assume we'll be able to link to the return value in the future? I guess that is something worth adding in the style guide.

Reply:
```suggestion
    Solve the equation ``a x = b`` for `x`, assuming `a` is a triangular matrix.
```